nifi-extension-bundles/nifi-poi-bundle/nifi-poi-services/src/main/java/org/apache/nifi/excel/ExcelHeaderSchemaStrategy.java

```diff
@@ -55,18 +52,18 @@ public class ExcelHeaderSchemaStrategy implements SchemaAccessStrategy {
 
     private final PropertyContext context;
     private final ComponentLog logger;
-    private final TimeValueInference timeValueInference;
+    private final CellFieldTypeReader cellFieldTypeReader;
     private final DataFormatter dataFormatter;
 
     public ExcelHeaderSchemaStrategy(PropertyContext context, ComponentLog logger, TimeValueInference timeValueInference, Locale locale) {
         this.context = context;
         this.logger = logger;
-        this.timeValueInference = timeValueInference;
+        this.cellFieldTypeReader = new StandardCellFieldTypeReader(timeValueInference);
         this.dataFormatter = locale == null ? new DataFormatter() : new DataFormatter(locale);
```

Comment:
I do not think the Locale argument is needed any longer as we can instantiate DataFormatter as follows:
```suggestion
        this.dataFormatter = new DataFormatter(Locale.getDefault());
```